.gitignore

```diff
@@ -1,3 +1,4 @@
 build
 .DS_Store
 *.o
+wgrib2/config.h
```

Comment:
OK, instead of being build in the code directory, config.h should be build in the build directory, and then the build directory has to be included in the list of include file dirs.

Reply:
Fixed